cypress/fixtures/docs/dynamicObjects.md

```diff
@@ -0,0 +1,275 @@
+| params                  | parameters of the method                                                                                  | {}                                            | string or object               |
+| validationJsonPath      | Contains an array or string with the path of the response value that needs to be validated                | result.enabled                                | string or array                |
+| setMethod               | The setter method name to make a call, associated with set test steps                                     | closedcaptions.setEnabled                     | string                         |
+| setParams               | Parameters of the set method to set new value.                                                            | {value: true}                                 | string or object               |
+| setValidationJsonPath   | Contains an array or string with the path of the response value that needs to be validated.               | result.enabled                                | string or array                |
+| setContent              | Holds the source of truth to validate against the response of the set method.                             | null                                          | string, null, boolean or array |
+| event                   | Name of the event to start listening for.                                                                 | accessibility.onClosedCaptionsSettingsChanged | string                         |
+| eventValidationJsonPath | Contains an array or string with the path of the response value that needs to be validated for the event. | result.enabled                                | string or array                |
+| content                 | Holds the source of truth to validate against the response of th get method or event.                     | true                                          | string, null, boolean or array |
+
+**Note:**
+
+- Acceptable types for the above params i.e string, number, array, object etc.
+- These are all the supported fields but depending on the test scenario and glue code used none of them are technically required.
+
+## Runtime Variables
+
+Runtime variables are used to store the values that are populated from the test case during execution.
+
+Below are the runtime variables that can be used in the fireboltCall object.
+
+- **attribute:** Attribute is a field name of the `runtime` environment variable that holds the value of the method name. For example, if the method name is `closedCaptions.setEnabled`, the attribute value will be `enabled`.
+- **value:** Value is a field name of the `runtime` environment variable that holds the value used for to set the value or for validation.
+
+**Note:** In order to use the runtime variables in the fireboltCall object, the `resolveAtRuntime()` function should be used.
+
```

Comment:
**Note:** In order to use runtime variables in fireboltCall object for api invocation, validation, FCS makes use of a function `resolveAtRuntime()` which is explained below